src/ResourceManager/HDInsight/Commands.HDInsight/JobCommands/GetAzureHDInsightJobOutputCommand.cs

```diff
@@ -133,25 +134,38 @@ private static string Convert(Stream stream)
 
         internal IStorageAccess GetDefaultStorageAccess(string resourceGroupName, string clusterName)
         {
+            var StorageAccountSuffix = "";
```

Comment:
Parsing the storage account is a standard way to obtain the suffix. This way we are not dependent on the test execution context and the same code can be ported elsewhere. Also, when the storageAccountSuffix is null or empty the SDK picks the default azure environment.